pwndbg/gdblib/disasm/__init__.py

```diff
@@ -454,8 +454,11 @@ def near(
                 emulated_arm_mode_cache[emu.pc] = emu.read_thumb_bit()
 
         # Handle visual splits in the disasm view
-        # The second check here handles instructions like x86 `REP` that repeat the instruction
-        if insn.has_jump_target or insn.next == insn.address:
+        # We create splits in 3 conditions:
```

Comment:
What exactly are splits? xD when we put an empty line to show there is a jump/nonlinear pc change or sth?